recipes/priori/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "priori" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/ohsu-comp-bio/regulon-enrichment/archive/refs/tags/v1.0.3.tar.gz
+  sha256: 12108f54ff41c1a2cd374154e3d95cbb3a9dbca277dc16f0db6b354245dbd939
+
+build:
+  number: 0
+  entry_points:
+    - priori=priori.priori:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python ==3.6.15
+    - dill ==0.3.1.1
+    - numpy ==1.17.3
+    - pandas ==0.25.3
+    - pip
+    - setuptools
+    - scikit-learn ==0.21.3
+    - scipy ==1.5.2
+    - tqdm ==4.38.0
+    - statsmodels ==0.12.2
+
+  run:
+    - python ==3.6.15
+    - dill ==0.3.1.1
+    - numpy ==1.17.3
+    - pandas ==0.25.3
+    - scikit-learn ==0.21.3
+    - scipy ==1.5.2
+    - tqdm ==4.38.0
+    - statsmodels ==0.12.2
```

Comment:
These pins are too tight! Down to the micro version!! This is bound to break. Can you check/test with upstream to why these are needed? I'm pretty sure they can be relaxed to a `>=` rather than this.

Reply:
Good to know. I just listed the package versions from the conda environment that I tested and ran Priori. I'll relax the pins as you suggested.